src/rgw/rgw_rest_s3.cc

```diff
@@ -3876,10 +3876,19 @@ void RGWSetRequestPayment_ObjStore_S3::send_response()
 
 int RGWInitMultipart_ObjStore_S3::get_params(optional_yield y)
 {
+  int ret;
+
+  map_qs_metadata(s, true);
```

Comment:
Is this call necessary?  The aws doc doesn't appear to document using query parameters to set encryption parameters when creating a multipart upload.

Reply:
Removed.